tools/scalar-schema/src/scalar_schema/dynamo/common.clj

```diff
@@ -18,8 +18,9 @@
 
 (defn- get-table-name-without-prefix
   [{:keys [prefix] :as schema}]
-  (str/replace-first (get-table-name schema)
-                     (java.util.regex.Pattern/compile (str prefix \_)) ""))
+  (let [table-name (get-table-name schema)]
+    (if prefix (str/replace-first table-name (java.util.regex.Pattern/compile (str prefix \_)) "")
+        table-name)))
```

Comment:
I think it can be more readable.
It is better to avoid making lines longer than 80 characters according to the [clojure-style-guide](https://github.com/bbatsov/clojure-style-guide).
```suggestion
    (if prefix
      (str/replace-first table-name
                         (java.util.regex.Pattern/compile (str prefix \_)) "")
      table-name)))
```